tests/planning_poker/test_consumers.py

```diff
@@ -1,22 +1,54 @@
 from unittest.mock import Mock, patch
 
 import pytest
+from channels.db import database_sync_to_async
+from channels.testing import WebsocketCommunicator
+
+from example.asgi import application
+from planning_poker.models import PokerSession, Story
 
 
 class TestPokerConsumer:
-    def test_poker_session(self, poker_consumer):
-        assert poker_consumer.poker_session.id == 1
-
-    # @pytest.mark.django_db
-    # @pytest.mark.asyncio
-    # async def test_connect(self):
-    #     application = URLRouter([
-    #         url(r'/(?P<poker_session>\w+)/$', PokerConsumer)
-    #     ])
-    #     communicator = WebsocketCommunicator(application, 'ws://test/planning_poker/1/')
-    #     connected, subprotocol = await communicator.connect()
-    #     assert connected
-    #     await communicator.disconnect()
+    def test_poker_session(self, poker_consumer, poker_session):
+        assert poker_consumer.poker_session.id == poker_session.id
+
+    @pytest.mark.asyncio
+    @pytest.mark.django_db
+    @pytest.mark.parametrize('has_active_story', [True, False])
+    async def test_connect(self, has_active_story):
+        poker_session = PokerSession(poker_date='2000-01-01', name='Test Sprint')
```

Comment:
You could simply use your existing fixtures `poker_session` and `story` here to avoid having to create new objects manually. The actual setup code would then only need to be:
```python
        if has_active_story:
            poker_session.active_story = story
            await database_sync_to_async(poker_session.save)()
```

For this to work, the DB mark must be changed to `@pytest.mark.django_db(transaction=True)`.